### PR TITLE
Github actions

### PR DESCRIPTION
**Pull request by josh13w**
Fix github actions for pushing changes to main and prd to firebase hosting live.

### CODE REVIEW

File: .github/workflows/firebase-hosting-prd-merge.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
-name: Deploy to Dev on Merge With Main
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on merge
```
Comment (benwulfe) [inline .github/workflows/firebase-hosting-prd-merge.yml:4]: why does this still say main if its for PRD?

**Review by benwulfe — CHANGES_REQUESTED** [review]
seems to want to deploy main to production?  we want prd...

File: .github/workflows/firebase-hosting-prd-merge.yml (hunk 1 above)
Comment (josh13w) [reply .github/workflows/firebase-hosting-prd-merge.yml]: Good catch

File: .github/workflows/firebase-hosting-prd-merge.yml [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
       - uses: actions/checkout@v3
-      - run: cd sample_tracking && npm run build && firebase use staging
+      - run: cd sample_tracking && npm install && npm run build
       - uses: FirebaseExtended/action-hosting-deploy@v0
-        with:
+       with:
           entrypoint: ./sample_tracking
           repoToken: '${{ secrets.GITHUB_TOKEN }}'
-          firebaseServiceAccount: '${{ secrets.FIREBASE_SERVICE_ACCOUNT_RIVER_SKY_386919 }}'
+          firebaseServiceAccount: '${{ secrets.FIREBASE_SERVICE_ACCOUNT_TIMBERID_PRD }}'
```
[the remark this answers is not in the document]
Comment (kazob1998) [reply .github/workflows/firebase-hosting-prd-merge.yml:19]: just double checked and it is there

**Review by kazob1998 — APPROVED** [review]

**Review by kazob1998 — APPROVED** [review]

**Review by benwulfe — APPROVED** [review]

**Pull request merged**